### ISSUE TITLE
Doesn't work with babel 7-alpha

### ISSUE THREAD

**Issue by hlehmann** [yahoo/babel-plugin-react-intl#122]
I'm using the new version of babel, but I have to disable babel-plugin-react-intl do to the following error:
```
Module build failed: file.has is not a function
    at PluginPass.pre (node_modules/babel-plugin-react-intl/lib/index.js:199:23)
```

**Comment by xtuc** [comment]
Do you mind opening an issue on the Babel repo? With more context, thanks.

**Comment by hlehmann** [comment]
This plugin still does not work on babel 7 beta. Did you try it ?
What context do you want ?

**Comment by Borales** [comment]
I'm having the same issue, using this boilerplate (https://github.com/kriasoft/react-starter-kit/tree/feature/react-intl - `feature/react-intl` branch).

**Comment by Borales** [comment]
I've submitted PR to Babel repo (https://github.com/babel/babel/pull/6529)

**Comment by okuryu** [comment]
Merged #129, then it still doesn't works?

**Comment by xtuc** [comment]
Babel 7 has not been released. You probably mean the alphas and betas?

**Comment by fbartho** [comment]
Sorry! I misspoke! yes, I meant the betas. I’m coming from react-native’s package.json which references "@babel/core": "7.0.0-beta.40",.  Instead of released, I should have said “depended on by a major project I don’t control”.

I’m getting closer to Release, so I’d like to launch with latest React-native if possible. I’d rather not have to downgrade my React-native in order to extract strings. Hope that usefully explains my motivations!

**Comment by tstirrat15** [comment]
What's the status on this? This is work I'd like to see go through as well.

**Comment by okuryu** [comment]
I just published `babel-plugin-react-intl@3.0.0`. Please try to test it, and feel free to reopen this issue or open a new issue if you have any problems.
